daily-challenges/2024-09/24/hunghoang3110.spec.ts

```
@@ -0,0 +1,38 @@
+// # Javascript
+function getCurrentDate() {
+    const curDate = new Date();
+    let day = curDate.getDate().toString();
+    let month = (curDate.getMonth() + 1).toString();
+    let year = curDate.getFullYear();
+    day = day.length < 2 ? '0' + day : day;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)